README.md

```diff
@@ -4,7 +4,12 @@ This is a seed project to get us started with prototyping.
 
 ## Installation
 
-First install the toolchain with [NPM](https://www.npmjs.org/):
+First install the command line tools globally with [NPM](https://www.npmjs.org/):
+```shell
+npm install --global gulp uglify-js2
```

Comment:
What specific error(s) did you get?

I'm definitely a fan of having setup instructions be *super* clear and accurate. I did a clean install test, but it's possible I had something installed globally that I didn't realize...

Point me at the error and we'll get it sorted out! :)